src/devices/BrickPi3/Movement/Motor.cs

```diff
@@ -176,18 +195,22 @@ private void OnPropertyChanged(string name)
         /// </summary>
         public event PropertyChangedEventHandler PropertyChanged;
 
-        private int periodRefresh;
+        private int _periodRefresh;
```

Comment:
Reflection breaking change here as well, but I'm fine to take it too. This one we probably don't even have to document.

Reply:
Yeh, I wouldn't worry especially those are not marked as serializable